### ISSUE TITLE
Code-coverage error on PHP 7.2

### ISSUE THREAD

**Issue by Minstel** [Codeception/Codeception#5392]
#### What are you trying to achieve?

Produce code coverage html reports. Command used:
`bin/codecept run unit --coverage-html -vvv`

Error happens for any suite, I'm showing unit-suite here as example.

#### What do you get instead?

Error message, after all the tests are finished:
```bash
In ErrorHandler.php line 83:

  [PHPUnit\Framework\Exception (2)]
  count(): Parameter must be an array or an object that implements Countable


Exception trace:
 () at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Subscriber\ErrorHandler.php:83
 Codeception\Subscriber\ErrorHandler->errorHandler() at n/a:n/a
 count() at D:\WebServers\home\workflow-engine.localhost\vendor\phpunit\php-code-coverage\src\Report\Html\Renderer\File.php:308
 SebastianBergmann\CodeCoverage\Report\Html\File->renderSource() at D:\WebServers\home\workflow-engine.localhost\vendor\phpunit\php-code-coverage\src\Report\Html\Renderer\File.php:61
 SebastianBergmann\CodeCoverage\Report\Html\File->render() at D:\WebServers\home\workflow-engine.localhost\vendor\phpunit\php-code-coverage\src\Report\Html\Facade.php:117
 SebastianBergmann\CodeCoverage\Report\Html\Facade->process() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Coverage\Subscriber\Printer.php:112
 Codeception\Coverage\Subscriber\Printer->printHtml() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Coverage\Subscriber\Printer.php:69
 Codeception\Coverage\Subscriber\Printer->printResult() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\event-dispatcher\EventDispatcher.php:212
 Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\event-dispatcher\EventDispatcher.php:44
 Symfony\Component\EventDispatcher\EventDispatcher->dispatch() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Codecept.php:209
 Codeception\Codecept->printResult() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Command\Run.php:405
 Codeception\Command\Run->execute() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\console\Command\Command.php:255
 Symfony\Component\Console\Command\Command->run() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\console\Application.php:901
 Symfony\Component\Console\Application->doRunCommand() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\console\Application.php:262
 Symfony\Component\Console\Application->doRun() at D:\WebServers\home\workflow-engine.localhost\vendor\symfony\console\Application.php:145
 Symfony\Component\Console\Application->run() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\src\Codeception\Application.php:108
 Codeception\Application->run() at D:\WebServers\home\workflow-engine.localhost\vendor\codeception\codeception\codecept:43

```
### Details

* Codeception version: 2.5.3
* PHP Version: 7.2.15
* Operating System: Windows 7
* Installation type: Composer
* List of installed packages (`composer show`)
```bash
$ composer show
behat/gherkin                       v4.6.0             Gherkin DSL parser ...
codeception/codeception             2.5.3              BDD-style testing f...
codeception/phpunit-wrapper         6.0.14             PHPUnit classes use...
codeception/stub                    2.0.4              Flexible Stub wrapp...
composer/xdebug-handler             1.3.2              Restarts a process ...
container-interop/container-interop 1.2.0              Promoting the inter...
desarrolla2/cache                   v3.0.0             Provides an cache i...
dflydev/apache-mime-types           v1.0.1             Apache MIME Types
doctrine/inflector                  v1.3.0             Common String Manip...
doctrine/instantiator               1.1.0              A small, lightweigh...
facebook/webdriver                  1.6.0              A PHP client for Se...
flow/jsonpath                       0.4.0              JSONPath implementa...
fzaninotto/faker                    v1.8.0             Faker is a PHP libr...
guzzlehttp/guzzle                   6.3.3              Guzzle is a PHP HTT...
guzzlehttp/promises                 v1.3.1             Guzzle promises lib...
guzzlehttp/psr7                     1.5.2              PSR-7 message imple...
hoa/compiler                        3.17.08.08         The Hoa\Compiler li...
hoa/consistency                     1.17.05.02         The Hoa\Consistency...
hoa/event                           1.17.01.13         The Hoa\Event library.
hoa/exception                       1.17.01.16         The Hoa\Exception l...
hoa/file                            1.17.07.11         The Hoa\File library.
hoa/iterator                        2.17.01.10         The Hoa\Iterator li...
hoa/math                            1.16.08.29         The Hoa\Math library.
hoa/protocol                        1.17.01.14         The Hoa\Protocol li...
hoa/regex                           1.17.01.13         The Hoa\Regex library.
hoa/stream                          1.17.02.21         The Hoa\Stream libr...
hoa/ustring                         4.17.01.16         The Hoa\Ustring lib...
hoa/visitor                         2.17.01.16         The Hoa\Visitor lib...
hoa/zformat                         1.17.01.10         The Hoa\Zformat lib...
improved/function                   dev-master c0c961e Function handling a...
improved/iterable                   dev-master a4b304a Functions to intera...
improved/type                       dev-master 964eb1f Type handling
jasny/application-env               v1.0.0             Logic around the AP...
jasny/autowire                      v1.2.0             Autowiring for PSR-...
jasny/codeception-module            dev-master 490b1cd Codeception Module ...
jasny/config                        v2.0.0             Configure your appl...
jasny/container                     1.2.1              Simple dependency i...
jasny/controller                    v1.2.1             A general controlle...
jasny/db                            v2.4.14            Adds modern OOP DB ...
jasny/db-mongo                      v1.3.0             Adds modern OOP DB ...
jasny/dotkey                        v1.0.2             Access objects and ...
jasny/error-handler                 v0.2.0             Error handler with ...
jasny/event-dispatcher              v0.1.0
jasny/http-message                  v1.3.5             PSR-7 implementatio...
jasny/meta                          v3.0.3             Define metadata for...
jasny/php-code-quality              v2.2.1             Quality assurance t...
jasny/php-functions                 v3.3.1             A set of useful PHP...
jasny/reflection-factory            v1.0.0             Jasny skeleton library
jasny/router                        v1.1.2             A versatile router ...
jasny/typecast                      v2.1.1             Type casting with b...
jasny/validation-result             v1.1.2             A result object for...
jean85/pretty-package-versions      1.2                A wrapper for ocram...
kornrunner/keccak                   1.0.3              Pure PHP implementa...
legalthings/base58-compat           v0.1.0             Userspace function ...
legalthings/data-enricher           v0.8.2             Enrich objects by p...
legalthings/lto-api                 dev-php7 b4e32a9   LegalThings One client
legalthings/mongodb-session-handler v2.0.0             A session handler t...
mikey179/vfsStream                  v1.6.5             Virtual file system...
mongodb/mongodb                     1.4.2              MongoDB driver library
monolog/monolog                     1.24.0             Sends your logs to ...
mtdowling/jmespath.php              dev-master a132511 Declaratively speci...
mustache/mustache                   v2.12.0            A Mustache implemen...
myclabs/deep-copy                   1.8.1              Create deep copies ...
nesbot/carbon                       2.11.0             A simple API extens...
nette/bootstrap                     v2.4.6             🅱 Nette Bootstra...
nette/di                            v2.4.15            💎 Nette Dependen...
nette/finder                        v2.4.2             🔍 Nette Finder: ...
nette/neon                          v2.4.3             🍸 Nette NEON: en...
nette/php-generator                 v3.2.1             🐘 Nette PHP Gene...
nette/robot-loader                  v3.1.0             🍀 Nette RobotLoa...
nette/utils                         v2.5.3             🛠 Nette Utils: l...
nikic/php-parser                    v4.2.0             A PHP parser writte...
ocramius/package-versions           1.3.0              Composer plugin tha...
overtrue/phplint                    1.1.9              a php syntax check ...
paragonie/random_compat             v2.0.18            PHP 5.x polyfill fo...
php-http/guzzle6-adapter            v1.1.1             Guzzle 6 HTTP Adapter
php-http/httplug                    v1.1.0             HTTPlug, the HTTP c...
php-http/promise                    v1.0.0             Promise used for as...
phpdocumentor/reflection-common     1.0.1              Common reflection c...
phpdocumentor/reflection-docblock   4.3.0              With this component...
phpdocumentor/type-resolver         0.4.0
phpspec/prophecy                    1.8.0              Highly opinionated ...
phpstan/phpdoc-parser               0.3.1              PHPDoc parser with ...
phpstan/phpstan                     0.10.8             PHPStan - PHP Stati...
phpstan/phpstan-strict-rules        0.10.1             Extra strict and op...
phpunit/php-code-coverage           5.2.0              Library that provid...
phpunit/php-file-iterator           1.4.5              FilterIterator impl...
phpunit/php-text-template           1.2.1              Simple template eng...
phpunit/php-timer                   1.0.9              Utility class for t...
phpunit/php-token-stream            2.0.2              Wrapper around PHP'...
phpunit/phpunit                     6.0.8              The PHP Unit Testin...
phpunit/phpunit-mock-objects        4.0.0              Mock Object library...
psr/container                       1.0.0              Common Container In...
psr/http-message                    1.0.1              Common interface fo...
psr/log                             1.1.0              Common interface fo...
psr/simple-cache                    1.0.1              Common interfaces f...
ralouphie/getallheaders             2.0.5              A polyfill for geta...
ramsey/uuid                         3.8.0              Formerly rhumsaa/uu...
respect/validation                  1.1.29             The most awesome va...
sebastian/code-unit-reverse-lookup  1.0.1              Looks up which func...
sebastian/comparator                1.2.4              Provides the functi...
sebastian/diff                      1.4.3              Diff implementation
sebastian/environment               2.0.0              Provides functional...
sebastian/exporter                  2.0.0              Provides the functi...
sebastian/global-state              2.0.0              Snapshotting of glo...
sebastian/object-enumerator         2.0.1              Traverses array str...
sebastian/recursion-context         2.0.0              Provides functional...
sebastian/resource-operations       1.0.0              Provides a list of ...
sebastian/version                   2.0.1              Library that helps ...
squizlabs/php_codesniffer           3.4.0              PHP_CodeSniffer tok...
stephenhill/base58                  v1.1.4             Base58 Encoding and...
symfony/browser-kit                 v4.2.3             Symfony BrowserKit ...
symfony/console                     v4.2.3             Symfony Console Com...
symfony/contracts                   v1.0.2             A set of abstractio...
symfony/css-selector                v4.2.3             Symfony CssSelector...
symfony/dom-crawler                 v4.2.3             Symfony DomCrawler ...
symfony/event-dispatcher            v4.2.3             Symfony EventDispat...
symfony/finder                      v4.2.3             Symfony Finder Comp...
symfony/polyfill-ctype              v1.10.0            Symfony polyfill fo...
symfony/polyfill-mbstring           v1.10.0            Symfony polyfill fo...
symfony/process                     v4.2.3             Symfony Process Com...
symfony/translation                 v4.2.3             Symfony Translation...
symfony/yaml                        v4.2.3             Symfony Yaml Component
theseer/tokenizer                   1.1.0              A small library for...
webmozart/assert                    1.4.0              Assertions to valid...
webmozart/glob                      4.1.0              A PHP implementatio...
webmozart/path-util                 2.3.0              A robust cross-plat...
willdurand/negotiation              v2.3.1             Content Negotiation...

```
* Suite configuration:

```yml
# Codeception Test Suite Configuration

# suite for unit (internal) tests.
class_name: UnitTester
modules:
    enabled: 
        - Asserts
        - \Helper\General
        - \Helper\Unit
coverage:
    enabled: true
    remote: false
    include:
        - lib/*
        - models/*
        - services/*
```

**Comment by Naktibalda** [comment]
Try updating your dependencies, you have an odd versions of phpunit and php-code-coverage installed that are easy to update to the latest patch release.
```
phpunit/phpunit                     6.0.8
phpunit/php-code-coverage           5.2.0
```

**Comment by Naktibalda** [comment]
@Minstel Update your dependencies to solve this issue.
It was fixed in phpunit/php-code-coverage 5.2.3

https://github.com/sebastianbergmann/php-code-coverage/commit/fa0ada5757b8f148be7c198af711b8365bfe89cc
